src/dune_util/report_error.ml

```diff
@@ -200,8 +200,13 @@ let gen_report exn backtrace =
       | User -> msg
       | Developer -> append msg (i_must_not_crash ())
     in
+    Log.info_user_message msg;
     Console.print_user_message msg
 ;;
 
 let report { Exn_with_backtrace.exn; backtrace } = gen_report exn (Some backtrace)
-let report_exception exn = gen_report exn None
+
+let report_exception exn =
+  let backtrace = Printexc.get_raw_backtrace () in
```

Comment:
This backtrace is probably wrong. We shouldn't capture it here.

Reply:
Still wrong

Reply:
Yes, I haven't touched it yet.